From fe3aaaeb1a27c4ef82397d6866a9900203d43e29 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 4 Jul 2016 23:08:43 -0400
Subject: [PATCH 2/2] do not support manually set time

Since NTP setting is not supported in anaconda, it does not
support set time manually neither.

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/ui/gui/spokes/datetime_spoke.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyanaconda/ui/gui/spokes/datetime_spoke.py b/pyanaconda/ui/gui/spokes/datetime_spoke.py
index 542c3f9..5d71d0e 100644
--- a/pyanaconda/ui/gui/spokes/datetime_spoke.py
+++ b/pyanaconda/ui/gui/spokes/datetime_spoke.py
@@ -510,6 +510,9 @@ class DatetimeSpoke(FirstbootSpokeMixIn, NormalSpoke):
                              _("Restoring hardware time..."))
             threadMgr.wait(constants.THREAD_TIME_INIT)
 
+        # Do not support manually set time
+        self._set_date_time_setting_sensitive(False)
+
         hubQ.send_ready(self.__class__.__name__, False)
 
     @property
-- 
2.8.1

